### DISCUSSION TITLE
about release mode enclave

### DISCUSSION THREAD

**Discussion by qsmen** [occlum/occlum#908] [General]
Hi,
     it is said that for production usage,  enclave shoud be in release mode, and should be checked and judged by a trusted client through remotely attesting the enclave.  But the process is so complex that I want to know why I choose release mode?
 what is the difference between debug mode and release mode?  In default, my debug mode enclave also return isvenclavequotestatus=ok. it seems that enclave works well.
 thank you.

**Comment by volcano0dr** [comment]
Enclave In debug mode, Host OS can use SGX instructions (`EDBGRD`, `EDBGWR`) to read and write enclave memory.

**Reply by qsmen** [reply]
thank you for the reply. I got it.
